packages/mui-material/src/LinearProgress/LinearProgress.d.ts

```diff
@@ -13,7 +13,8 @@ export interface LinearProgressProps
    */
   classes?: Partial<LinearProgressClasses>;
   /**
-   * The color of the component. It supports those theme colors that make sense for this component.
+   * The color of the component. It supports those default theme colors that make sense for this component,
+   * or custom theme colors, defined as shown in the [palette customization guide](https://mui.com/customization/palette/#adding-new-colors).
```

Comment:
@samuelsycamore This is the current iteration of the prop description. It hasn't been propagated to the js or json files, so I suspect you might be commenting about the previous iteration?

Reply:
@samuelsycamore to be honest I forgot about this PR, but would be great if you can double check so I can replicate this on all places necessary and close it :)

Reply:
Sorry @mnajdova I completely forgot about this one, too! 😅 Can you clarify what is meant by "those theme colors that make sense for this component"? How would I, as a user, know what "makes sense" for the component in this context?

Reply:
I think we can remove the 
> "It supports those theme colors that make sense for this component,"

in favor of 

> "It supports the default theme colors,"

as by default it supports all colors we have in the default MD theme: 'primary', 'secondary', 'error', 'info', 'success' and 'warning'.